src/api/app/services/workflows/step_iterator.rb

```diff
@@ -0,0 +1,26 @@
+module Workflows
```

Comment:
I don't see the point in having this class. It's increasing complexity.

Reply:
No, it's not, in fact it's helping me reducing the complexity of `Token::Workflow#call` method. CodeClimate whined about it before.

Reply:
It's a whole new file to only check if all workflows are valid by running `raise (...) workflows.reject(&:valid?).any?`. Everything else is boilerplate code.

Reply:
Okay, let me see what I can do